src/markdown-pages/highlights.md

```diff
@@ -12,7 +12,19 @@ Post a question or share your experiences with other users in our <a href="https
 Access the Galasa source code in [GitHub](https://github.com/galasa-dev) and open issues in the [project management repository](https://github.com/galasa-dev/projectmanagement).
 
 
-## 0.26.0 - Release Highlights
+## 0.27.0 - Release Highlights
+
+- REST API enhancements:<br><br>
+       * allow queries of test run details using the run name<br><br>
+       * documentation added to the https://rest.galasa.dev/ website<br><br>
+
+- Command-line tool `galasactl` now supports:<br><br>
+       * the `GALASA_HOME` environment variable, which you can set to avoid using the `{HOME}/.galasa` folder <br><br>
+       * the `--development` flag for `galasactl project create` and `galasactl local init` to enable generated code to use pre-released code versions of Galasa libraries, in addition to the ones published on Maven central <br><br>
+       * automated tests run on every build <br>
+
```

Comment:
galasactl now also supports `galasactl runs get --runname` to get details of a specific run from an ecosystem.

To specify the ecosystem bootstrap location, you can pass in the `--bootstrap` flag (e.g. `--bootstrap https://<your-ecosystem-url>/bootstrap`).

Reply:
thanks Eamonn. As discussed, going to add that detail in a different PR around viewing test results and keep the headline of querying test run details on run name for the highlights.